tbconnect/models.py

```diff
@@ -130,6 +157,10 @@ def get_processed_data(self):
             "language": self.language,
         }
 
+    def update_group_arm(self):
+        if not self.group_arm:
```

Comment:
It might also be a good idea to put this behind a feature flag from settings, then we can switch it on/off using env variables.